## Welcome to my page!

Hello world! My name is Carmelle. I am PhD student studying RNA Splicing in Computational Biology at the University of California Berkeley. 
![image](https://user-images.githubusercontent.com/67977500/178340607-ba324fe7-6572-4604-9590-0bab227d1774.png)


### About Me

I grew up in the small island-province of Cebu, Philippines. When I was fifteen years-old I relocated to San Jose, California when I migrated to the United States. I attended college at the University of California Santa Cruz where I earned my bachelor's degree in Biomolecular Engineering & Bioinformatics. When I'm not doing science, I enjoy beautiful hikes, [running](https://www.strava.com/athletes/99557318), enjoying good food with friends, taking care of my plants or exploring a new artsy hobby!

Feel free to follow me in my science journey on [twitter](https://twitter.com/carmeile) or contact me at carmelle@berkeley.edu :-)


### Research Projects

**ALLELE-SPECIFIC SPLICING EDITING FOR HUNTINGTON’S DISEASE**

_Lareau Lab University of California, Berkeley_

My work with the Innovative Genomics Institute here at UC Berkeley is exploring the use of single-nucleotide variations (SNVs) to anchor base editors that induce splicing changes. We use machine learning to evaluate splicing changes in our targets computationally. Using this strategy, we can target disease that are most resistant to CRISPR editing: diseases caused by repeat expansion mutations.

**MUTUALLY EXCLUSIVE SPLICING ANALYSIS (MESA) (WINTER 2019 – SPRING 2020)**

_Brooks Lab University of California, Santa Cruz_

Mutually Exclusive Splicing Analysis (MESA) is a tool that identifies alternatively spliced junctions through mutually exclusive intron clusters. Currently, MESA does not identify retained intron events. For this project, I added caller for retained introns to MESA. I evaluated the performance of this feature by comparing intron retention calls that were made by other tools (JuncBASE, MAJIQ and JUM) on RNA-Seq data from HeLa cells that were treated with spliceosome-inhibiting drugs

**DETECTION AND EVALUATION OF INTRON RETENTION EVENTS (FALL 2019 – SPRING 2020)**

_Brooks Lab and Jurica Lab University of California, Santa Cruz_

As part of a collaborative project between Brooks Lab and Jurica Lab, I ran 20 samples of HeLa cells that have been treated with varying amounts of different spliceosome inhibitors through four different pipelines that identify alternative splicing events to evaluate how treatment of spliceosome-inhibiting drugs affects intron retention.

**ANALYSIS OF EPIGENETIC DYSREGULATION IN KAT6A MUTANT PATIENTS (SUMMER 2019)**

_Arboleda Lab University of California, Los Angeles_

As part of the B.I.G. Summer Research Program, I conducted research in the Arboleda lab investigating the KAT6A mutation in patient fibroblast cells. The KAT6A gene encodes for an enzyme that acetylates histones. Because mutations in this gene disrupts normal acetylation, we hypothesize that the KAT6A mutation results in a cascade of epigenetic dysregulation of other epigenetic markers. For this project, I helped develop a similarity test based on set theory to recursively build histone features and find overlapping regions between these combinatorial features and differentially methylated sites in KAT6A mutants. Our results suggest that differential methylation in KAT6A mutants are enriched at H2AFZ, H3K9me3 histone mark, and sites of active transcription. Additionally, we found hypomethylated sites enriched in binding sites of EZH2 and RNF2, members of PRC1 and PRC2, suggesting a possible connection between KAT6A and the PRC complexes.

**ABERRANT SPLICING AS BIOMARKERS IN CANCER CELL LINES SENSITIZED TO DRUGS THAT TARGET SPLICING REGULATORS (FALL 2018 – JUNE 2019)**

_Brooks Lab University of California, Santa Cruz_

We hypothesize that aberrant splicing in cancer cell lines contributed to sensitivity towards drugs that target splicing regulators. In this project, we developed our own method of identifying aberrant splicing events and used publicly available pharmacological data to test our hypothesis. From my analysis using the Broad Cancer Cell Line Encyclopedia pharmacological data, I found hematopoietic and lymphatic cancer cell lines to be sensitized to the drug, Palbociclib. This finding was exciting to us for two main reasons. First, Palbociclib is a kinase inhibitor; We were particularly interested in kinase inhibitors as they may affect splicing directly. Second, our collaborator has identified Palbociclib to be weakly active in splicing through an exon-skipping assay in his preprint (Shi et al. 2019). From my analysis using pharmacological data from the Genomics of Drug Sensitivity in Cancer, I found a correlation between aberrant splicing in some cancer cell lines and nine kinase-targeting drugs, suggesting their potential role in splicing and possible therapeutic applications.


**KRAS MUTATION ASSOCIATED ALTERNATIVE SPLICING (SPRING 2017)**

_Brooks Lab University of California, Santa Cruz_

Because mutated forms of the KRAS gene are oncogenic drivers to various types of carcinomas, our hypothesis was that KRAS mutations affected splicing in other genes as well. For this project, I analyzed alternative splicing data on KRAS mutations in LUAD patients. I helped develop a downstream analysis and visualization tool for JuncBASE in order to visualize data from KRAS mutant patients. The data analysis we performed using our tool concluded that the KRAS mutation is strongly associated with differential splicing in 120 genes.


### Relevant Experience

**JUNCBASE OPTIMIZATION (SPRING-SUMMER 2017)**

_Brooks Lab University of California, Santa Cruz_

The goal of this project is to optimize JuncBASE , a bioinformatics tool for RNA-seq alignment reads to identify and quantify alternative splicing events. One major issue addressed were unplaced scaffolds that caused errors in later steps in the pipeline.

**LOW-COST ELECTROCARDIOGRAPH (WINTER 2017)**

_Dr. Kevin Karplus, University of California, Santa Cruz_

The goal of this project was to create a low-cost electrocardiograph using a microcontroller called TeensyBoard and PteroDAQ, an open-source data-acquisition system. This project was done as a part of a freshman design course.

